resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -241,6 +244,56 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class Scrollbar {
+
+    [DscProperty(Key)] [Bool] $Show = $false
+
+
+
+    hidden [string] $DynamicScrollbarProperty = 'DynamicScrollbar'
+
+    [Scrollbar] Get() {
+
+        $currentState = [AlwaysShowScrollbar]::new()
+        
+		if (-not(DoesRegistryKeyPropertyExist -Path $global:ControlPanelAccessibilityRegistryPath -Name $this.DynamicScrollbarProperty)) {
+            $currentState.Show = $false        
+        } else {
+			$dynamicScrollbarValue = (Get-ItemProperty -Path $global:ControlPanelAccessibilityRegistryPath -Name $this.DynamicScrollbarProperty).DynamicScrollbar
+			$currentState.Show = ($dynamicScrollbarValue -eq 0) ? $true : $false
+
+				
+		}
+		
+        return $currentState
+    }

```

Comment:
Nit: There is something still wrong with the alignment of this.

Reply:
This is a resurgence of the `tab v space` war of ~2008, and the truce was to set tabs to 4 spaces, which is why this looks normal in my IDE. GitHub is not honoring the truce.